src/docfx/publish/PublishModelBuilder.cs

```diff
@@ -43,7 +43,6 @@ public bool TryAdd(Document file, PublishItem item)
             var monikers = item.Monikers;
             if (monikers.Count == 0)
             {
-                // TODO: report a warning if there are multiple files published to same url, one of them have no version
                 monikers = new List<string> { "NONE_VERSION" };
```

Comment:
suggest to replace this magic string `NONE_VERSION` with a boolean flag indicating whether this file has versions